docs/generated_code.md

```diff
@@ -27,8 +27,8 @@ If only dealing with a single file,  run `scripts/generate_source.py` with `--ta
 
 ```bash
 # Example - only generates chassis.h
-scripts/generate_source.py external/Vulkan-Headers/registry/ --target vk_layer_dispatch_table.h
+scripts/generate_source.py external/Vulkan-Headers/registry/ --target vul_dispatch_table.h
 ```
 
 When making change to the `scripts/` folder, make sure to run `generate_source.py`
-(Code generation does **not** happen automatically at build time.)
\ No newline at end of file
+(Code generation does **not** happen automatically at build time.)
```

Comment:
```suggestion
Code generation does **not** happen automatically at build time. To make the library more consumable.
```

Reply:
I think theres been enough force pushes for *one* PR :stuck_out_tongue:

Reply:
Yeah that's a FFTI no worries

Reply:
> To make the library more consumable.

Also thats a sentence fragment. what is making the library more consumable? Should just drop the period.